download_contributors_avatars.py

```diff
@@ -0,0 +1,48 @@
+# -*- coding: utf-8 -*-
+"""
+Created on Mon Apr  8 18:09:49 2019
+
+@author: Nataliya_Pavych
+"""
+
+import argparse
+import requests
+import os
+import bs4
+
+
+def download_avatar(directory, request_response):
+    with open(os.path.join(f'{directory}', os.path.basename(url)), 'wb') as image_file:
+        for chunk in request_response.iter_content(chunk_size=1024):
+            image_file.write(chunk)
+    
+            
+def extract_download_avatar(full_url, directory):
+    response = requests.get(full_url)
+    if response.status_code == 200:
+        soup = bs4.BeautifulSoup(response.content, 'html.parser')
+        web_object = soup.select('img')
+        for image_number in range(len(web_object)):
+            image_url = web_object[image_number].get('src')
+            if ['.jpg', '.png'] not in str(image_url):

```

Comment:
I think that the URLs do not actually end with jpg/png. The issue may be related to this.